client/my-sites/stats/feedback/index.tsx

```diff
@@ -127,16 +128,18 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 	const [ isOpen, setIsOpen ] = useState( false );
 	const [ isFloatingPanelOpen, setIsFloatingPanelOpen ] = useState( false );
 
+	const { isCommercialOwned } = useStatsPurchases( siteId );
```

Comment:
Using `isCommercialOwned` limits us to only those sites with a commercial Stats plan. It does not include Business or Complete plans so it's the narrowest version of the target audience.

If we want to include those plans, we can update this to use the `supportCommercialUse` value instead. That value covers Stats plans as well as Business and Complete plans.

Reply:
I prefer using `supportCommercialUse` as those users can experience the same functionalities. @kangzj What do you think?

Reply:
I don't have a strong opinion. I would go for `supportCommercialUse` because it's not a massive user base and the audience in this group should be all able to use all of Jetpack Stats and might have feedback to share with us.

Reply:
Sounds good! I'll update it to `supportCommercialUse` along with hiding the inline card.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/94525